python/hawkey/goal-py.cpp

```diff
@@ -149,6 +150,8 @@ args_run_parse(PyObject *args, PyObject *kwds, int *flags, PyObject **callback_p
         *flags |= DNF_IGNORE_WEAK_DEPS;
     if (ignore_weak)
         *flags |= DNF_IGNORE_WEAK;
+    if (no_protected)
+        *flags |= DNF_NO_PROTECTED;
```

Comment:
What about to use ``DNF_IGNORE_PROTECTED``?

Reply:
Probably lets wait litlebit with remaining. I would like to suggest other changes in the code, therefore there could be change in meaning of the flag.